build/README.md

```diff
@@ -8,34 +8,50 @@ The build system is tested on Linux (x86_64 and arm64) and macOS (Intel chip and
 
 ## Prerequisites
 
+The following examples should be performed under the Kong source codebase.
+
 The build system requires the following tools to be installed:
 
 - [Bazel/Bazelisk](https://bazel.build/install/bazelisk), Bazelisk is recommended to ensure the correct version of Bazel is used.
-    - Use `make check-bazel` to install Bazelisk into `bin/bazel`, then use `export PATH=bin:$PATH` to add it into your `PATH`.
-- [Build dependencies](https://github.com/Kong/kong/blob/master/DEVELOPER.md#build-and-install-from-source)
 
+  We can install Bazelisk by running the following command:
+
+  ```bash
+    // install Bazelisk into $PWD/bin/bazel
+    make check-bazel
+    // add Bazelisk into your $PATH
+    export PATH=bin:$PATH
+    // check bazel version
+    bazel version
+  ```
+
+- [Build dependencies](https://github.com/Kong/kong/blob/master/DEVELOPER.md#build-and-install-from-source)
 
 ## Building
 
-To build Kong and all its dependencies, run the following command:
+### Build Kong and its dependencies
+
+The commands related to building Kong are already supported in the [Makefile](../Makefile), so we can use them directly:
 
 ```bash
-bazel build //build:kong --verbose_failures
+make build-kong
```

Comment:
maybe let's still keep the `bazel build //build:kong --verbose_failures` command here as a `equivelant` option